samples/apps/forum/src/authentication.ts

```diff
@@ -10,23 +12,104 @@ export interface User {
     userId: string
 }
 
+interface HeaderClaims {
+    kid: string
+}
+
+interface BodyClaims {
+    sub: string
+    iss: string
+}
+
+interface MSAccessTokenClaims extends BodyClaims {
+    aud: string
+    appid: string // 1.0 only
+    ver: string // 1.0 or 2.0
+}
+
+const MS_APP_ID = "1773214f-72b8-48f9-ae18-81e30fab04db"
```

Comment:
A comment with a link explaining how to get your own MS_APP_ID may be worthwhile here. I suggest https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/walkthrough-register-app-azure-active-directory